File: decompiled/sources/com/hjq/http/request/OptionsRequest.java
package com.hjq.http.request;

import androidx.lifecycle.LifecycleOwner;
import com.hjq.http.model.HttpMethod;

public final class OptionsRequest extends UrlRequest<OptionsRequest> {
    public OptionsRequest(LifecycleOwner lifecycleOwner) {
        super(lifecycleOwner);
    }

    /* access modifiers changed from: protected */
    public String getRequestMethod() {
        return HttpMethod.OPTIONS.toString();
    }
}

File: decompiled/sources/com/hjq/http/request/OptionsRequest.kt
package com.hjq.http.request

import androidx.lifecycle.LifecycleOwner
import com.hjq.http.model.HttpMethod

class OptionsRequest(
    lifecycleOwner: LifecycleOwner
) : UrlRequest<OptionsRequest>(lifecycleOwner) {
    override fun getRequestMethod(): String {
        return HttpMethod.OPTIONS.toString()
    }
}
